### DISCUSSION TITLE
好像本人 cookie 可以查到天赋等级？

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#762] [General]
还没仔细看具体实现
https://github.com/CMHopeSunshine/LittlePaimon#%E5%8A%9F%E8%83%BD%E7%A4%BA%E4%BE%8B

**Comment by mark9804** [comment]
https://github.com/CMHopeSunshine/LittlePaimon/blob/026f46a3ae02bf9381d56df401475aa1c5ecc0f2/hoshino/modules/Genshin_Paimon/get_data.py#L68-L81

草，还能找出这种 API

**Reply by Arondight** [reply]
确实不是一个画风

**Reply by mark9804** [reply]
这应该是刚开服的某一个活动相关的 API ，但我完全没有印象有哪个活动能查看角色天赋等级

**Reply by Arondight** [reply]
可能是废案忘了关了，这个日期没有活动

**Comment by Xm798** [comment]
确实可以

![image](https://user-images.githubusercontent.com/48074861/160653950-be02f788-0dee-4579-b21b-18391357118d.jpeg)
